apps/mastodoncounter/mastodon_counter.star

```diff
@@ -0,0 +1,222 @@
+        if GET_MASTODON.status_code != 200:
+            return connectionError()
+        GET_FOLLOWERS = GET_MASTODON.json()["followers_count"]
+        GET_USERNAME = GET_MASTODON.json()["username"]
+        finalusername = str(GET_USERNAME)
+        cache.set("cached_name", finalusername, ttl_seconds = 900)
+        removedecimal = int(GET_FOLLOWERS)
+        extraone = removedecimal + +1
+        followercount = str(removedecimal)
+        cache.set("cached_count", followercount, ttl_seconds = 900)
+        plusone = str(extraone)
+        cache.set("cached_plusone", plusone, ttl_seconds = 900)
+
+    if config.bool("pronoun"):
+        thepronoun = "We're"
+    else:
+        thepronoun = "I'm"
+
+    if config.bool("scrolly"):
+        themessage = followercount + " and counting! Want to be number " + plusone + "? " + thepronoun + " @" + finalusername + "@" + stringserver
+    else:
+        themessage = followercount
+
+    return render.Root(
+        delay = 0,
+        child = render.Column(
+            children = [
+                render.Box(
+                    width = 64,
+                    height = 18,
+                    color = "#6364ff",
+                    child = render.Row(
+                        expanded = True,
+                        children = [
+                            render.Image(
+                                width = 18,
+                                height = 16,
+                                src = MASTODON_ICON,
+                            ),
+                            render.WrappedText("MASTODON FOLLOWERS"),
+                        ],  #children
+                    ),  #render.Row
+                ),  #render.Box
+                render.Box(
+                    width = 64,
+                    height = 14,
+                    color = "#000000",
+                    child = render.Row(
+                        expanded = True,
+                        children = [
+                            render.Marquee(
+                                width = 64,
+                                offset_start = 64,
+                                child = render.Text(
+                                    font = "6x13",
+                                    content = themessage,
+                                ),
+                            ),
+                        ],  #children
+                    ),  #render.Row
+                ),  #render.Box
+            ],  #children
+        ),  #render.Column
+    )  #render.Root
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "instance",
+                name = "Your Mastodon instance",
+                desc = "The Mastodon server you use, e.g. ohai.social or mastodon.social",
+                icon = "mastodon",
+            ),
+            schema.Text(
+                id = "apikey",
+                name = "Your access token",
```

Comment:
The `desc` field is currently not visible to users of the Tidbyt iOS/Android app, so be aware that any information users will need to set up the app has to be in the `name` field, or in the app description (in `mastodoncounter.go`).